server/command.go

```diff
@@ -261,6 +261,14 @@ func (p *Plugin) runHelpCommand(user *model.User) (string, error) {
 }
 
 func (p *Plugin) runSettingCommand(args *model.CommandArgs, params []string, user *model.User) (string, error) {
+	if _, authErr := p.authenticateAndFetchZoomUser(user); authErr != nil {
+		// the user state will be needed later while connecting the user to Zoom via OAuth
+		if appErr := p.storeOAuthUserState(user.Id, args.ChannelId, false); appErr != nil {
+			p.API.LogWarn("failed to store user state")
+		}
+		return authErr.Message, authErr.Err
+	}
+
```

Comment:
What's the message that's shown here?

Reply:
User is shown a link to connect the account if it is not connected. It is the same flow as when the user tries to start meeting while not being connected https://github.com/mattermost/mattermost-plugin-zoom/blob/ae9b9248a081c507dbab9c8de66a912c760871cc/server/command.go#L128